#goal definition
geometry_msgs/PoseStamped in_wall_brick_pose
---
#result definition
string message
---
#feedback